package.json

```diff
@@ -35,7 +37,8 @@
     "react-scripts": "^4.0.3",
     "redux": "^4.0.5",
     "redux-thunk": "^2.3.0",
-    "uuid": "^3.4.0"
+    "uuid": "^3.4.0",
+    "yup": "^0.32.9"
```

Comment:
I think validation will be a bigger issue to tackle to I'm not sure yup will be an adequate long-term solution. Looks like this was discussed in this [PR](https://github.com/Lambda-School-Labs/frontend-vbb-portal/pull/61)

We may want to hold off on adding validation for a future feature.

Reply:
Thanks for bringing this PR to my attention. We'll take a look at it.